lib/matplotlib/tests/test_mathtext.py

```diff
@@ -343,6 +343,16 @@ def test_operator_space(fig_test, fig_ref):
     fig_ref.text(0.1, 0.8, r"$\mathrm{log}_2$")
 
 
+@check_figures_equal(extensions=["png"])
+def test_genfrac_displaystyle(fig_test, fig_ref):
+    fig_test.text(0.1, 0.1, r"$\dfrac{2x}{3y}$")
+
+    thickness = _mathtext.TruetypeFonts.get_underline_thickness(
+        None, None, fontsize=12, dpi=mpl.rcParamsDefault['figure.dpi'])
```

Comment:
I think that should be rcParams, not rcParamsDefault (in theory, tests could configure a different rcParams dpi)

Reply:
Ah, so that's what I was missing. Thanks for the heads-up.